documentation/docs/pages/operators/nodes/validator-setup/nym-api.mdx

```diff
@@ -1,28 +1,36 @@
+import { Callout } from 'nextra/components';
```

Comment:
Getting the following error for all validator files: 

```
./pages/operators/nodes/validator-setup.mdx:12:0
Module not found: Can't resolve '@nextui-org/react'
  10 | import {VarInfo} from 'components/variable-info.tsx';
  11 | import {Steps} from 'nextra/components';
> 12 | import {Accordion, AccordionItem} from "@nextui-org/react";
  13 | export const __toc = [{
  14 |   depth: 2,
  15 |   value: "Building your validator",
```

Reply:
Did you install the component locally? For me it works

But I can remove the component for now